### ISSUE TITLE
[BUG] - Address validation error

### ISSUE THREAD

**Issue by serderovsh** [tronprotocol/tron-web#117]
@Kondax, Have you noticed that when validating an address, sometimes an error climbs, although the logic must be "false".

Example
```javascript
    const isValid = tronWeb.isAddress('THwdafY3TEsuEbrRBSQ7fbvPxiVBcYdoi0');
    console.log(isValid); // Error: Non-base58 character

    const isValid = tronWeb.isAddress('THwdafY3TEsuEbrRBSQ7fbvPxiVBcYdoi1');
    console.log(isValid); // False
```
If at the end of the address 0 an error is displayed

**Comment by serderovsh** [comment]
It is possible, if you have any other options.
```javascript
        try {
            return utils.crypto.isAddressValid(address);
        }catch (e) {
            return false;
        }
```

**Comment by Kondax** [comment]
I have experienced this, I'll get a push fixed soon.

**Comment by sullof** [comment]
I addressed this problem in the PR for the test coverage, fixing a lot of other minor issues. I haven't merged it, right now, because I am not sure that it does not introduce some conflict. But you can take a look
https://github.com/tronprotocol/tron-web/pull/91

**Comment by serderovsh** [comment]
@sullof, But I did something like this in my project. It is written in TypeScript, in any case I needed to recreate the method of address verification.

Thanks!.

**Comment by serderovsh** [comment]
The problem is solved #91
